docs/docs/explanations/advanced-configuration.md

```diff
@@ -283,19 +283,32 @@ domain: demo.nebari.dev
 
 ### Continuous integration and continuous deployment
 
-Nebari uses [infrastructure-as-code](https://en.wikipedia.org/wiki/Infrastructure_as_code) to allow developers and users to request changes to the environment via pull requests (PRs) which then get approved by administrators.
-You may configure a CI/CD process to watch for pull-requests or commits on specific branches.
-Currently, CI/CD can be setup for either [GitHub Actions](https://docs.github.com/en/actions) or [GitLab CI](https://docs.gitlab.com/ee/ci/).
+Nebari uses [infrastructure-as-code](https://en.wikipedia.org/wiki/Infrastructure_as_code) to declare a desired state for the deployment.  By using a git repository with CI/CD configured, teams can more quickly modify their deployment, empowering developers and data scientists to request the changes and have them approved by an administrator.
 
+When a `ci_cd` section is configured within your `nebari-config.yaml`, the `nebari render` command will create or update pipeline definition files which describe a CI/CD process to redeploy Nebari when changes are made to a specified branch.  Currently, Nebari can generate CI/CD for [GitHub Actions](https://docs.github.com/en/actions) and [GitLab CI](https://docs.gitlab.com/ee/ci/).
```

Comment:
The use of `render` is not well streamlined to users as we want to reduce the level of abstraction as much as possible when deploying (render is called in the background before the stages begin). 

I would suggest rephrasing this to say that `deploy` will create the necessary files during the first time Nebari is deployed. And include a note saying that if the user wants to "update" the files as if running a dry-run, it can use the render.

Reply:
Updated